server/src/test/java/io/spine/server/procman/given/ProcessManagerRepositoryTestEnv.java

```diff
@@ -169,19 +177,66 @@ CommandRouted handle(PmStartProject command, CommandContext context) {
                     .routeAll();
         }
 
-        @Subscribe
-        void on(EntityAlreadyArchived rejection) {
+        @React
+        Empty on(EntityAlreadyArchived rejection) {
             keep(rejection);
+            return withNothing();
         }
 
-        @Subscribe
-        void on(EntityAlreadyDeleted rejection) {
+        @React
+        Empty on(EntityAlreadyDeleted rejection) {
             keep(rejection);
+            return withNothing();
         }
 
         @Override
         public String getIdString() {
             return getId().toString();
         }
     }
+
+    public static class GivenCommandMessage {
+
+        public static final ProjectId ID = Sample.messageOfType(ProjectId.class);
+
+        /** Prevents instantiation on this utility class. */
```

Comment:
We have this formatting for all the utility constructors. This way it's compact.